python/ray/serve/backend_worker.py

```diff
@@ -411,7 +412,10 @@ async def handle_request(self,
             self.replica_tag, request.metadata.request_id, request_time_ms))
 
         self.num_ongoing_requests -= 1
-        return result
+        # Returns a small object for router to track request status.
+        # The self.num_ongoing_requests is chosen because it's small and may
+        # be useful for the router.
+        return self.num_ongoing_requests, result
```

Comment:
Since we're not using this information I'd prefer to minimize serialization cost by passing an empty byte string to bypass the serialization codepath (`b""`)